IFC4x3/Properties/b/BerthTonnage_3mukLKUYvFDBZX5V9O8L21/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BerthTonnage_3mukLKUYvFDBZX5V9O8L21" Name="BerthTonnage" UniqueId="f0e2e554-7a2e-4f34-b8e1-15f258215081" PrimaryDataType="IfcMassMeasure" />
```

Comment:
I guess same comment as the other tonnage. Does this imply a unit?

Reply:
i see your point, renamed for unit nutraility